src/common/constants.ts

```diff
@@ -14,12 +16,16 @@ export const SERVICES: Record<string, symbol> = {
   METER: Symbol('Meter'),
   APPLICATION: Symbol('Application'),
   ELASTIC_CLIENTS: Symbol('ElasticClients'),
+  REDIS: Symbol('Redis'),
   S3_CLIENT: Symbol('S3Client'),
   CLEANUP_REGISTRY: Symbol('CleanupRegistry'),
 };
 /* eslint-enable @typescript-eslint/naming-convention */
 
 export const ON_SIGNAL = Symbol('onSignal');
 export const HEALTHCHECK = Symbol('healthcheck');
+export const REDIS_SYMBOL = Symbol('REDIS');
 
+export const REDIS_TTL = 300;
 export const elasticConfigPath = 'db.elastic';
+export const siteIndex = 1;
```

Comment:
The schema of the s3 endpoint urls that we use are divided using '.'  when the ENDPOINT_URL[1] is the site we're currently on.

Reply:
Because this constant is not common and only one file is using this constant variable, you should move it to the head of file that is using this variable.